app/controllers/application_controller.rb

```diff
@@ -30,6 +31,13 @@ def after_sign_out_path_for(resource_or_scope)
     end
   end
 
+  def set_active_banner
+    return nil unless current_organization
+
+    @active_banner = current_organization.banners.active.first
+    @active_banner = nil if session[:dismissed_banner] == @active_banner&.id
```

Comment:
Should this session variable actually hold an array of dismissed banners? Or does it not really matter since we can only have one active banner at a time.

Reply:
There's only one active banner at once so I think it's 50-50 on whether to store an array of IDs or just the single one. Since storing an array in session storage is simple but non-trivial, I'd opt for the single ID until we need otherwise.